### PR TITLE
feat: Connecting FaceId to Django [GG-38]

### PR DESCRIPTION
**Pull request by lb-cake**
- Implements necessary functions and APIs to allow the FaceID code to access and update information in the Django database.

### CODE REVIEW

**Comment by orndorffc** [comment]
Tested - Ready to Merge

**Comment by mraynor** [comment]
There are 3 commits with almost the same name: 'Update testfile', 'Update test file in FaceID' and 'Update test file', could these be reduced to one commit?

File: GGProject/workflow/apps.py [hunk 1]
```diff
@@ -3,4 +3,4 @@
 
 class WorkflowConfig(AppConfig):
     name = 'workflow'
-    verbose_name = 'Objects'
+    verbose_name = 'Objects'
```
Comment (kesslerk) [inline GGProject/workflow/apps.py]: This file has no new line at the end. Thats what that red icon is showing. Its pretty standard to have a new line at the end of files. I believe it's also a readability thing

File: GGProject/workflow/tests.py [hunk 2]
```diff
@@ -2,9 +2,8 @@
...
 from rest_framework.views import status
-from .models import Employee
-from .serializers import EmployeesSerializer
+from .models import Employee, Picture, TempPhoto
+from .serializers import EmployeesSerializer, PicturesSerializer, TempPhotosSerializer
 from . import views
 
-
-# Place your tests here
+# Place your tests here
```
Comment (kesslerk) [inline GGProject/workflow/tests.py]: Same for this file (no new line at end)

**Comment by kesslerk** [comment]
We have to make sure all the migration files that are auto generated are not included once the .gitignore merge request is merged

File: GGProject/workflow/migrations/0011_picture_name.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    dependencies = [
+        ('workflow', '0010_auto_20190618_1349'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='picture',
+            name='name',
+            field=models.CharField(default='cat.jpg', max_length=30),
+        ),
```
Comment (mraynor) [inline GGProject/workflow/migrations/0011_picture_name.py]: Trailing comma?

**Comment by kemptonk** [comment]
The migrations have been cleared so that should not be an issue anymore.

File: GGProject/GreeterGuru/urls.py [hunk 4]
```diff
@@ -18,5 +18,6 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    re_path('api/', include('workflow.urls')),    
+    #re_path('api/(?P<version>(v1|v2))/', include('workflow.urls')),
+    path('api/', include('workflow.urls')),
```
Comment (kesslerk) [inline GGProject/GreeterGuru/urls.py:22]: The last line of an array should not end in a comma

File: GGProject/workflow/urls.py [hunk 5]
```diff
@@ -2,8 +2,9 @@
...
 urlpatterns = [
-    path('employees/', views.ListEmployeesView.as_view(), name="employees-all"),
-    path('pictures/', views.ListPicturesView.as_view(), name="pictures-all"),
-    path('temp-photos/', views.ListTempPhotosView.as_view(), name="temp-photos-all"),
-
+    path('employees/', views.ListEmployees),
+    path('employees/<int:emp_ID>/', views.SingleEmployee),
+    path('pictures/', views.ListPictures),
+    path('pictures/<int:emp_ID>/', views.EmployeePictures),
+    path('pictures/<str:name>/', views.SinglePicture),
```
Comment (kesslerk) [inline GGProject/workflow/urls.py:9]: The last line of an array should not end in a comma

Comment (kemptonk) [reply GGProject/workflow/urls.py:9]: That's how all of the default Django code does it, and also how all of its example documentation does it.

Comment (kesslerk) [reply GGProject/workflow/urls.py:9]: Ok that's fine

**Review by kesslerk — APPROVED** [review]
👍

**Pull request merged**